x-pack/plugins/reporting/server/lib/export_types_registry.ts

```diff
@@ -91,7 +88,7 @@ export class ExportTypesRegistry {
   }
 }
 
-function getExportTypesRegistryFn(): ExportTypesRegistry {
+export function getExportTypesRegistry(): ExportTypesRegistry {
```

Comment:
The export type registry gets referenced in ReportingCore, which gets set up 1 time, so this module does not need to return a singleton.